Kotlin/graphql-simple-bindings-emitter/src/main/kotlin/community/flock/graphqlsimplebindings/emitter/KotlinEmitter.kt

```diff
@@ -12,10 +12,10 @@ class KotlinEmitter(private val packageName: String = "community.flock.graphqlsi
     override fun emitDocument(document: Document): String = super.emitDocument(document)
             .let { "package $packageName\n\n$it" }
 
-    override fun ObjectTypeDefinition.emitObjectTypeDefinition() = "data class $name(\n${fieldDefinitions.emitFields()}\n)\n"
+    override fun ObjectTypeDefinition.emitObjectTypeDefinition() = "data class $name(\n${fieldDefinitions.emitDefinitionFields()}\n)\n"
```

Comment:
What's the difference between a Field and a DefinitionField?

Reply:
I renamed emitFields to emitDefinitionFields because of namen conflicts